configure.ac

```diff
@@ -9,11 +9,11 @@ dnl Licensed under Apache License v2.0.  See the file "LICENSE" for more
 dnl information.
 dnl
 
-dnl We need at least autoconf 2.60...
-AC_PREREQ([2.60])
+dnl We need at least autoconf 2.71...
+AC_PREREQ([2.71])
 
 dnl Package name and version...
-AC_INIT([CUPS], [2.4.0], [https://github.com/openprinting/cups/issues], [cups], [https://openprinting.github.io/cups])
+AC_INIT([CUPS],[2.4.0],[https://github.com/openprinting/cups/issues],[cups],[https://openprinting.github.io/cups])
```

Comment:
Any reason for removing the spaces?

Reply:
autoupdate side effect

Reply:
@AtariDreams Never use autoupdate on CUPS, it can break things... :/